Makefile

```diff
@@ -9,20 +9,16 @@ PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 all: manager
 
 # Current Operator version
-VERSION ?= 0.0.1
+VERSION ?= 0.8.0
 # Default bundle image tag
 BUNDLE_IMG ?= controller-bundle:$(VERSION)
 # Options for 'bundle-build'
-ifneq ($(origin CHANNELS), undefined)
-BUNDLE_CHANNELS := --channels=$(CHANNELS)
-endif
-ifneq ($(origin DEFAULT_CHANNEL), undefined)
-BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
-endif
+BUNDLE_CHANNELS := --channels="alpha,stable"
+BUNDLE_DEFAULT_CHANNEL := --default-channel=stable
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/3scale/apicast-operator:master
+IMG ?= quay.io/3scale/apicast-operator:v0.8.0
```

Comment:
Because we are using the 0.8.0 tag here, we need to tag 2.14 with 0.8.0 in order to get the image. Then test it (fresh and upgrade) and then possibly merge the PR.

Reply:
Tag -  after PR merge. We do test on PR's Branch before merge.
Release test can be done after merge and tagging - suppose need separate Jira for this. Need discuss with Tomas and Brian, with Team. It's a process of Community Release testing. I will send the question in common chat.
Thank you for comment @MStokluska